### PR TITLE
Ts 879

### PR DESCRIPTION
**Pull request by Toru8P**

### CODE REVIEW

File: src/main/java/com/exactpro/th2/sim/impl/Simulator.java [hunk 1]
```diff
@@ -96,22 +99,31 @@ public void init(@NotNull AbstractCommonFactory factory) throws Exception {
...
-                String sessionAlias = message.getMetadata().getId().getConnectionId().getSessionAlias();
-                if (StringUtils.isNotEmpty(sessionAlias)) {
-                    try {
-                        handleMessage(sessionAlias, message);
-                    } catch (Exception e) {
-                        logger.error("Can not handle message = {}", TextFormat.shortDebugString(message), e);
+            for (MessageGroup messageGroup: batch.getGroupsList()) {
+                for (AnyMessage anyMessage : messageGroup.getMessagesList()) {
+
+                    if (anyMessage.getKindCase().equals(AnyMessage.KindCase.RAW_MESSAGE)) {
```
Comment (ivandruzhinin) [inline src/main/java/com/exactpro/th2/sim/impl/Simulator.java:107]:
```suggestion
                    if (anyMessage.getKindCase() == AnyMessage.KindCase.RAW_MESSAGE) {
```

File: src/main/java/com/exactpro/th2/sim/impl/SimulatorRuleInfo.java [hunk 2]
```diff
@@ -157,18 +173,25 @@ public void send(@NotNull Message msg, long delay, @NotNull TimeUnit timeUnit) {
     }
 
     @Override
-    public void send(@NotNull MessageBatch batch, long delay, @NotNull TimeUnit timeUnit) {
```
Comment (ivandruzhinin) [inline src/main/java/com/exactpro/th2/sim/impl/SimulatorRuleInfo.java:160]: We can keep this method for backward compatibility, but mark it as deprecated

**Review by cordwelt — CHANGES_REQUESTED** [review]

File: src/main/java/com/exactpro/th2/sim/impl/SimulatorRuleInfo.java (hunk 2 above)
Comment (Topru333) [reply src/main/java/com/exactpro/th2/sim/impl/SimulatorRuleInfo.java:160]: true, thx for good idea

File: gradle.properties [hunk 3]
```diff
@@ -11,9 +11,9 @@
 # limitations under the License.
 ################################################################################
 
-release_version=4.0.0
+release_version=5.0.0
```
Comment (ivandruzhinin) [inline gradle.properties:14]:
```suggestion
release_version=4.1.0
```

**Review by cordwelt — CHANGES_REQUESTED** [review]

**Review by cordwelt — APPROVED** [review]

**Pull request merged**